OJP/OJP_Locations.xsd

```diff
@@ -45,7 +45,11 @@
 					<xs:documentation>If the response returns less results than expected, the value of skip can be used in a follow-up request to get further results. It tells the server to skip the given number of results in its response.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="PlaceResult" type="PlaceResultStructure" minOccurs="0" maxOccurs="unbounded"/>
+			<xs:element name="PlaceResult" type="PlaceResultStructure" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>The place / location object results found by the service. They have to be sorted by the ranking how well they match to the input data in descending order. The first result in the list matches best.</xs:documentation>
```

Comment:
> "They have to be sorted by the ranking how"
This does not make clear who sorts this and when. Maybe better:
"The place/location objects found by the service are ordered in descending order of how well they match the input data. The first result in the list matches best"